docs/release.rst

```diff
@@ -14,6 +14,12 @@ Unreleased
     # .. warning::
     #    Pre-release! Use :command:`pip install --pre zarr` to evaluate this release.
 
+* Fix bug that caused double counting of groups in ``groups()`` and ``group_keys()``
+  methods with V3 stores.
+  By :user:`Ryan Abernathey <rabernat>` :issue:`1228`.
+
+.. _release_2.13.2:
```

Comment:
This might end up being redundant:

https://github.com/zarr-developers/zarr-python/blob/main/docs/release.rst#2132

Reply:
I can't tell if you are requesting changes to my PR or not. Could you be more explicit?

Reply:
Sorry. I updated the branch and there doesn't look to be a conflict. (My expectation was there were going to be two)